app/models/forms/family_member.rb

```diff
@@ -181,7 +181,8 @@ def assign_person_address(person)
             next
           end
           if current_address.present?
-            person.addresses.update(address.permit!)
+            current_address.update(address.permit!)
```

Comment:
Good Catch 👍 . This allows address to be updated properly and also allow to flow this changes to Glue